gradle/instrumentation-library.gradle

```
@@ -15,7 +15,9 @@ dependencies {
   // TODO(anuraaga): We currently include common instrumentation logic like decorators in the
   // bootstrap, but we need to move it out so manual instrumentation does not depend on code from
   // the agent, like Agent.
-  api project(':auto-bootstrap')
+  api project(':auto-bootstrap'){
+    exclude group: 'org.slf4j', module: 'slf4j-simple'
+  }
 
```

```suggestion
  api deps.opentelemetrySdk
```
Would it make sense to add this dependency or are we giving user's the flexibility to supply their own sdk

---

i think leave it to users, since they have to pull in sdk to configure exporters anyways